lib/3llo/presenter/help.rb

```diff
@@ -1,13 +1,21 @@
 module Tr3llo
   module Presenter
+    FLAVOUR_HELP_INDENTED = 1
+    FLAVOUR_HELP_CLI = 2
+
     class HelpPresenter
-      def initialize(interface)
+      def initialize(interface, flavour = FLAVOUR_HELP_INDENTED)
         @interface = interface
+        @flavour = flavour
       end
 
       def print!
         interface.print_frame do
-          interface.puts menu_text
+          text = menu_text
+          if @flavour == FLAVOUR_HELP_CLI
```

Comment:
Instead of adding `flavour`, I think we could just add a parameter called `print!(stripped? = false)`?